src/components/shared/Button/PaneToggle.css

```diff
@@ -1,18 +1,21 @@
 .toggle-button-start,
 .toggle-button-end {
-  position: absolute;
-  width: 16px;
-  height: 16px;
   transition: transform 0.25s ease-in-out;
-  margin: 0 4px;
   cursor: pointer;
+  align-self: flex-end;
+  padding: 4px 2px;
 }
 
 .toggle-button-start svg,
 .toggle-button-end svg {
+  width: 14px;
   fill: var(--theme-comment);
 }
 
+.toggle-button-end {
+  margin-left: auto;
```

Comment:
I think this might be redundant with the `align-self` property from https://github.com/devtools-html/debugger.html/pull/1976/files#diff-65320c92508ade610814346c22bc028cR5